lib/basedriver/commands/find.js

```diff
@@ -273,29 +272,33 @@ helpers.findByImage = async function findByImage (b64Template, {
     }
   }
 
-  if (!rect) {
+  if (results.length === 0) {
     if (multiple) {
       return [];
     }
     throw new errors.NoSuchElementError();
   }
 
-  log.info(`Image template matched: ${JSON.stringify(rect)}`);
-  if (b64Matched) {
-    log.info(`Matched base64 data: ${b64Matched.substring(0, 200)}...`);
-  }
-  const imgEl = new ImageElement(b64Template, rect, score, b64Matched);
+  const elements = results.map((result) => {
+    log.info(`Image template matched: ${JSON.stringify(result.rect)}`);
+    const b64Matched = result.visualization;
+    if (b64Matched) {
+      log.info(`Matched base64 data: ${b64Matched.substring(0, 200)}...`);
```

Comment:
there is no need to log b64-encoded stuff

Reply:
This was existing behaviour; I've switched it to debug only but can remove.

Reply:
I would say we should not log base64 data itself. It only takes space without giving anything useful

Reply:
ok, removed